packages/terra-horizontal-divider/tests/jest/HorizontalDivider.test.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import HorizontalDivider from '../../src/HorizontalDivider';
+
+describe('HorizontalDivider', () => {
+  const defaultRender = <HorizontalDivider />;
+
+  // Snapshot Tests
+  it('should render a basic component', () => {
+    const wrapper = shallow(defaultRender);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  // Custom Prop Test
+  it('it should pass in a custom prop', () => {
+    const wrapper = shallow(<HorizontalDivider id="testDivider" />);
+    expect(wrapper.prop('id')).toEqual('testDivider');
```

Comment:
Test this by comparing snapshots. That way you'll be testing that the id is actually rendered on the element, not just that it was received as a prop.

Reply:
Updated https://github.com/cerner/terra-core/pull/962/commits/4e293135bbddadfbbbe2ee7da18c936300bbd2a4#diff-46429ad4c25a0b00f25c0054ce0dddf6R14